.github/workflows/tests.yml

```diff
@@ -44,8 +44,12 @@ jobs:
       - name: Prepare environment
         run: |
           sudo apt-get -y install libsqlite3-dev libxslt1-dev
-          gem uninstall bundler -v '>= 2' -ax --force || true
-          gem install bundler -v '~>1.0' -N || true
+          if [[ "${{ matrix.ruby }}" =~ ^2\.[4-6]$ ]]; then
+            gem uninstall bundler -v '>= 2' -ax --force || true
+            gem install bundler -v '~>1.0' -N || true
+          else
+            gem install bundler -v '>= 2'
+          fi
```

Comment:
It seems hacky to uninstall bundler and install it manually.
Could you try to use the `bundler: 1` or `bundler: 2` option of `ruby/setup-ruby`?
See https://github.com/ruby/setup-ruby/#bundler

Reply:
You can also try `bundler: none` if you don't want `ruby/setup-ruby` to ensure bundler is installed.

Reply:
Maybe it's related to the fact 2.7 ships Bundler 2.1.4 as a default gem, but the latest Bundler 2 is also version 2.1.4 and so:
```
$ gem list | grep bundler
bundler (default: 2.1.4)

$ gem install bundler
Fetching bundler-2.1.4.gem
Successfully installed bundler-2.1.4
1 gem installed

$ gem list | grep bundler
bundler (2.1.4)
```